addon-test-support/ember-test-helpers/legacy-0-6-x/ext/rsvp.js

```diff
@@ -11,13 +12,15 @@ let originalAsync;
   @private
 */
 export function _setupPromiseListeners() {
```

Comment:
Do we actually need to still call these as a noop (effectively) if you're < 1.7? Can't we just simply not call these at all?

Reply:
Ya, we could, but this seemed to localize the logic to the specific area that needed to care about the versioning.